modules/ztp-high-security-issues-addressed.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-security-hardening.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ztp-addressed-security-issues_{context}"]
+= Security hardening measures implemented
+
+== Disabling USB boot, wireless and bluetooth access  
+
+To prevent unauthorized modifications to the operating system and to reduce the risk of physical attack vectors, USB boot, wireless, and Bluetooth are disabled in the BIOS settings. These setting are not controlled through ZTP configuration. For example, disabling USB boot prevents attackers from booting from a USB drive and gaining unauthorized access to the system. The recommended firmware configuration is described in "Configuring host firmware for low latency and high performance".
```

Comment:
```suggestion
To prevent unauthorized modifications to the operating system and to reduce the risk of physical attack vectors, USB boot, wireless, and Bluetooth are disabled in the BIOS settings. These settings are not controlled by using ZTP configuration. Disabling USB boot prevents attackers from booting from a USB drive and gaining unauthorized access to the system. The recommended firmware configuration is described in "Configuring host firmware for low latency and high performance".
```

Reply:
Suggest removing "for example" because it doesn't seem to be an example of how these settings are not controlled by using ZTP configuration.